### ISSUE TITLE
What should the standard say for changes listed in Annex C that are backported by implementations?

### ISSUE THREAD

**Issue by cpplearner** [cplusplus/draft#6247]
Some changes are listed in Annex C [[diff]](http://eel.is/c++draft/diff) as differences between versions of C++, but implementations choose the new behavior even in old standard modes. Examples are:

* All 4 changes listed in [[diff.cpp20.utilities]](http://eel.is/c++draft/diff#cpp20.utilities)
* [[diff.cpp14.dcl.dcl]/2](http://eel.is/c++draft/diff#cpp14.dcl.dcl-2)
* [[diff.cpp14.class]/1](http://eel.is/c++draft/diff#cpp14.class-1)
* [[diff.cpp14.utilities]/1](http://eel.is/c++draft/diff#cpp14.utilities-1)

Should there be a note saying that implementations sometimes apply these changes in old standard modes?

Should these changes be removed from Annex C?

**Comment by jwakely** [comment]
No, they should not be removed. They describe differences between C++ standards, and so should be succulents in the standard. Those differences don't disappear just because implementations backport them. 

The behaviour of implementations should be documented by implementations, not in the standard.

**Comment by cpplearner** [comment]
I'd expect a brief note that describes the reality. Otherwise readers could be misled.

**Comment by jwakely** [comment]
Users are not reading the annexes of the standard

**Comment by jwakely** [comment]
These are informational notes about changes between revisions of ISO standards. If you want to know about real world implementation status, look elsewhere.

**Comment by jensmaurer** [comment]
Agreed with @jwakely ; the standard is not the manual for your implementation. I think the phrasing in the Annex is clear that it discusses differences between releases of the standard.

**Comment by jensmaurer** [comment]
There are no defect reports for C++, since we release a new standard every three years (that, officially, is unrelated to any previous standard).

We would have Defect Reports if we would publish a TC (Techincal Corrigendum).
